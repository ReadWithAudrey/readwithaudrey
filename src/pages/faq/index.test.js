import React from 'react'
import renderer from 'react-test-renderer'
import FAQ from './index.js'

describe('FAQ page snapshot test', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<FAQ />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
