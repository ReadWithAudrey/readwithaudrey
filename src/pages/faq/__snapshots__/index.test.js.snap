// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAQ page snapshot test renders correctly 1`] = `
Array [
  <header
    className="sc-bdVaJa flex items-center justify-between ph3 montserrat white bg-green w-100 f3 chiVga"
  >
    <div
      className="flex items-center"
      id="header-left"
    >
      <img
        alt="book icon"
        className="h2 ph2"
        src="test-file-stub"
      />
      <Link
        to="/"
      >
        Audrey
      </Link>
    </div>
    <Link
      className="w-third"
      to="/Form1/"
    >
      <button
        className="sc-gzVnrw bg-green hover-bg-pink pa2 mv2 white br2 w-100 f5 montserrat iQUxnM"
      >
        Join Us
      </button>
    </Link>
  </header>,
  <div
    className="flex flex-column items-center justify-center page-padding ph4"
  >
    <h1
      className="f2 pink tc montserrat mb3 mt4"
    >
      FAQs
    </h1>
    <h2
      className="f3 pink tc montserrat mb3"
    >
      What to expect
    </h2>
    <p
      className="sc-fjdhpX montserrat f5 tc lh-copy jcPxQN"
    >
      The Audrey experience comprises of five separate ten minute reading sessions. We choose a surprise book for you and divide it into 5 chapters to correspond with the five reading sessions. There’s a real mix of how regularly community members meet to read together. Some reading partners do one regular session each week, and therefore read the book over the course of about a month. Others read together over consecutive evenings and complete the book within a week. It’s really up to you and your reading partner. What we do encourage is that you only read one chapter during each session – even if it’s tempting to read on.
    </p>
    <p
      className="sc-fjdhpX montserrat f5 tc lh-copy jcPxQN"
    >
      Our community spend about ten minutes reading together, and then anything from just a couple of minutes right up to an hour chatting. This usually starts with a conversation about the text, and moves on to wider topics covering almost anything.
    </p>
    <p
      className="sc-fjdhpX montserrat f5 tc lh-copy jcPxQN"
    >
      Most people feedback and tell us that the intimacy of reading together helps them to feel at ease with their partner and allows them to build a deep and meaningful social connection together. Almost everyone who tries Audrey feeds back saying the experience helps them to feel relaxed. Our community often find themselves engaged in conversations with their reading partner that they might not usually have with the people in their day to day lives.
    </p>
    <h2
      className="f3 pink tc montserrat mb3"
    >
      Before the reading sessions
    </h2>
    <p
      className="sc-fjdhpX montserrat f5 tc lh-copy jcPxQN"
    >
      People can choose to be either the reader or listener, or have a go at both.
    </p>
    <p
      className="sc-fjdhpX montserrat f5 tc lh-copy jcPxQN"
    >
      It’s highly unlikely you’ll know the person you are partnered with, but we’ve attempted to pair readers and listeners who we think will connect well and enjoy each other’s company.
    </p>
    <p
      className="sc-fjdhpX montserrat f5 tc lh-copy jcPxQN"
    >
      We will introduce you to each other over email. We’ll share bios, provide contact details, and assist with arranging a time for the first reading session. We will provide details about the book you’ll be reading, so that you both have an idea of what to expect before the reading session.
    </p>
    <p
      className="sc-fjdhpX montserrat f5 tc lh-copy jcPxQN"
    >
      We will make sure you both receive a copy of the book (usually in a digital format) before the first reading session. This is usually attached to the email when we introduce you to your reading partner.
    </p>
    <h2
      className="f3 pink tc montserrat mb3"
    >
      The reading sessions
    </h2>
    <p
      className="sc-fjdhpX montserrat f5 tc lh-copy jcPxQN"
    >
      You only need a spare 10 minutes to have a reading session. We think Mondays are the best day to start reading sessions. In fact we think Mondays are the best day for all reading sessions. We gently encourage this habit within the Audrey community. Monday's Audrey Day! It's helpful for many, but it's not practical for everyone. Therefore we suggest Tuesdays as back up. If this isn’t possible either, then we kick you out.
    </p>
    <p
      className="sc-fjdhpX montserrat f5 tc lh-copy jcPxQN"
    >
      Reading sessions can take place over the phone (audio only) or you can use a messaging app (e.g. Whatsapp, Facebook Messenger or Apple Facetime) where you’ll have access to audio and video. You can decide this with your reading partner once you’ve been introduced over email.
    </p>
    <p
      className="sc-fjdhpX montserrat f5 tc lh-copy jcPxQN"
    >
      For each session you can expect the reading to take 10 minutes. Afterwards you’re welcome to open up conversations about the reading, but this is entirely up to you. There’s complete flexibility here - some users chat for a couple of minutes, some for up to an hour... some only talk about their thoughts on the book ... others dive into topics loosely connected to the story... often users end up talking about topics completely unrelated to the book.
    </p>
    <p
      className="sc-fjdhpX montserrat f5 tc lh-copy jcPxQN"
    >
      Don’t forget to agree the time for your next session before you say goodbye.
    </p>
    <h2
      className="f4 pink tc montserrat mb3"
    >
      After the reading sessions
    </h2>
    <p
      className="sc-fjdhpX montserrat f5 tc lh-copy jcPxQN"
    >
      We hope that the experience is mutually enjoyable and the book interesting and thought-provoking. It would be great to receive some feedback. Positives and negatives. Anything you enjoyed or did not enjoy about it. Ways that we could improve the experience etc.
    </p>
    <p
      className="sc-fjdhpX montserrat f5 tc lh-copy jcPxQN"
    >
      If you want to read another book, either with the same person, or someone else, please let us know. We’ll be delighted to organise this for you.
    </p>
    <p
      className="sc-fjdhpX montserrat f5 tc lh-copy jcPxQN"
    >
      Any questions, please get in touch.
    </p>
  </div>,
  <footer
    className="sc-bwzfXH flex flex-column justify-center items-center montserrat bg-green w-100 bFHdpU"
  >
    <span
      className="sc-htpNat flex items-center justify-around bg-green w-100 jbutSu"
    >
      <Link
        className="no-underline white ph1"
        to="/"
      >
        Home
      </Link>
      <Link
        className="no-underline white ph1"
        to="/story/"
      >
        Our Story
      </Link>
      <Link
        className="no-underline white ph1"
        to="/faq/"
      >
        FAQs
      </Link>
      <Link
        className="no-underline white ph1"
        to="/contact/"
      >
        Contact Us
      </Link>
    </span>
    <span
      className="sc-bxivhb flex justify-around items-center bg-white black w-100 cHkvlR"
    >
      © Audrey
      <Link
        className="no-underline black ph1"
        to="/Terms/"
      >
        Terms
      </Link>
      <Link
        className="no-underline black ph1"
        to="/Privacy/"
      >
        Privacy
      </Link>
    </span>
  </footer>,
]
`;
